MyString.java

```diff
@@ -18,13 +18,38 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+        String newStr = "";
+        for (int i = 0; i < str.length(); i++) {
+            char currentChar = str.charAt(i);
+            if ( currentChar >= 'A' && currentChar <= 'Z'){
+                newStr += (char) (currentChar + 32);
```

Comment:
Sometimes `+ 32` isn't clear enough.. 
I would rather you use `currentChar  - 'A' + 'a'` it will be clearer